automatic/supertuxkart/supertuxkart.nuspec

```diff
@@ -16,20 +16,15 @@ SuperTuxKart is a Free 3D kart racing game; we want to make the game fun more th
 See the great lighthouse or drive through the sand and visit the pyramids. Race underground or in space, watching the stars passing by. Have some rest under the palms on the beach (watching the other karts overtaking you :) ). But don't eat the bananas! Watch for bowling balls, plungers, bubble gum and cakes thrown by opponents.
 
 You can do a single race against other karts, compete in one of several Grand Prix, try to beat the high score in time trials on your own, play battle mode against your friends, and more!
-
-### Note
-
-Don’t close the empty appearing PowerShell window. Otherwise the silent installation won’t work.
     </description>
     <summary>Open Source 3D kart racing game</summary>
     <releaseNotes />
     <copyright />
-    <tags>supertuxkart racing game admin</tags>
+    <tags>supertuxkart foss cross-platform racing game admin</tags>
     <dependencies>
-      <dependency id="openal" />
-      <dependency id="vcredist2013" />
+      <dependency id="vcredist2015" />
```

Comment:
yeah, it would. But quoting the guidelines
> 3. When taking a dependency on anything else, specify minimum or exact version.

And personally, I think a minimum version should be specified (even if it really isn't needed)